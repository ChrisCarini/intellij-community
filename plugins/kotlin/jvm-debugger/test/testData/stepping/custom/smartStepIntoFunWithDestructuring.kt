fun main() {
    // SMART_STEP_INTO_BY_INDEX: 1
    // RESUME: 1
    //Breakpoint!
    foo(("key" to "value"))
}

fun foo(pair: Pair<String, String>) {
    val (key, value) = pair
    println("foo $pair")
}
