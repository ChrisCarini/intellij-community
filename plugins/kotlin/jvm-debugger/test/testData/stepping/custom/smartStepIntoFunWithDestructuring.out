LineBreakpoint created at smartStepIntoFunWithDestructuring.kt:5
Run Java
Connected to the target VM
smartStepIntoFunWithDestructuring.kt:5
smartStepIntoFunWithDestructuring.kt:9
Disconnected from the target VM

Process finished with exit code 0
